docs/DeveloperGuide.md

```diff
@@ -658,13 +658,13 @@ Manage contacts and schedule of students faster than a typical mouse/GUI driven
 
 **Extensions**
 
-- 1a. Teacher requests to find by address without providing any query.
-    - 1a1. Teacher’s Pet displays invalid address command message.
+- 1a. Teacher requests to find by address with a blank or white space only parameter.
+    - 1a1. Teacher’s Pet displays invalid address format command message.
```

Comment:
Should this be

Teacher requests to find by address with a blank or white space as the only parameter.